app/controllers/to_do_lists_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ToDoListsController < ApplicationController
+  def index
+    @lists = List.includes(:items).all
```

Comment:
Hey @vlasiak, let's do not expose ActiveRecord query interface here, but instead will base on the interface of the list:
```ruby
List.including_items
```
where `including_items` would be a scope.